crates/sui-indexer-alt-jsonrpc/src/api/transactions/filter.rs

```diff
@@ -394,11 +397,28 @@ async fn from_sequence_numbers(
         .transpose()
         .context("Failed to encode next cursor")?;
 
-    let digests = ctx
-        .pg_loader()
-        .load_many(rows.iter().map(|&seq| TxDigestKey(seq as u64)))
-        .await
-        .context("Failed to load transaction digests")?;
+    // Get digests from the table and retry if any of the digests are not found.
+    let retries = ctx.config().transactions.tx_digest_retry_count;
+    let retry_interval =
+        Duration::from_millis(ctx.config().transactions.tx_digest_retry_interval_ms);
+    let mut digests: HashMap<TxDigestKey, StoredTxDigest> = HashMap::new();
+    for _ in 0..retries {
```

Comment:
I thought about using a library like `backoff` or `tokio-retry` but they don't provide the functionalities I need (retry for a given number of times with custom error handling)  and only make the code longer, so I ended up just using a loop.

Reply:
A plain loop is fine, but it's better to use `tokio::time::interval` rather than `sleep` (see suggestion above).